osu.Game/Overlays/Settings/Sections/Online/AlertsAndPrivacySettings.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Configuration;
+
+namespace osu.Game.Overlays.Settings.Sections.Online
+{
+    public class AlertsAndPrivacySettings : SettingsSubsection
+    {
+        protected override string Header => "Alerts and Privacy";
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            Children = new Drawable[]
+            {
+                new SettingsCheckbox
+                {
+                    LabelText = "Show a notification popup when someone says your name",
+                    Bindable = config.GetBindable<bool>(OsuSetting.ChatHighlightName)
+                },
+                new SettingsCheckbox
+                {
+                    LabelText = "Show chat message notifications",
```

Comment:
isn't it more like "private message notifications"?

Reply:
Taken from stable. I'll be happy to change that.